pages/references.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center mb-6">Project References</h1>
+    <p class="mb-3">
```

Comment:
we might be able to remove styling like this where we have to add a bottom margin to a paragraph tag if we add in the typography plugin as noted in another PR. I might be wrong though. If not, is there a way we can style paragraphs without needing to manually add margins to each p tag? Would be nice if we can do this globally really

Reply:
have made global using tailwind in the layout files and then removed all the class="mb-3" across the pages that used it.

Reply:
change to just in basic layout as this is what the text pages use and was affecting the mapfilters

Reply:
Thanks Ruby. Looks good